content/ko/docs/contribute/new-content/overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 새로운 콘텐츠 기여하기에 대한 개요
+linktitle: 개요
+content_template: templates/concept
+main_menu: true
+weight: 5
+---
+
+{{% capture overview %}}
+
+이 섹션에는 새로운 콘텐츠를 기여하기 전에 알아야 할 정보가 있다.
+
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 기여하기에 대한 기본
+
+- 마크다운(markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
+- 소스는 [GitHub](https://github.com/kubernetes/website)에 있다. 쿠버네티스 문서는 `/content/en/docs/` 에서 찾을 수 있다. 일부 참조 문서는 `update-imported-docs/` 디렉터리의 스크립트에서 자동으로 생성된다.
```

Comment:
```suggestion
- 소스는 [GitHub](https://github.com/kubernetes/website)에 있다. 쿠버네티스 문서는 `/content/ko/docs/` 에서 찾을 수 있다. 일부 참조 문서는 `update-imported-docs/` 디렉터리의 스크립트에서 자동으로 생성된다.
```
Alternatively, mention both the English originals and the localized Korean versions.

Reply:
Yes, I also missed this point. I'll change it, too.